sdk/identity/Azure.Identity/src/AccessTokenUtilities.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Globalization;
+using System.IO;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+
+namespace Azure.Identity
+{
+    internal static class AccessTokenUtilities
+    {
+        public static async Task<AccessToken> DeserializeAsync(bool async, Stream content, CancellationToken cancellationToken)
```

Comment:
You can avoid making this method async if you use the stirng overload of Parse https://docs.microsoft.com/en-us/dotnet/api/system.text.json.jsondocument.parse?view=netcore-3.1#System_Text_Json_JsonDocument_Parse_System_String_System_Text_Json_JsonDocumentOptions_